hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmMetadataGenerator.java

```diff
@@ -157,6 +167,9 @@ enum Operation {
   private Operation[] operations;
   private boolean mixedOperation = false;
 
+  private final ReplicationConfig ratisThree =
+      ReplicationConfig.fromProtoTypeAndFactor(ReplicationType.RATIS, ReplicationFactor.THREE);
```

Comment:
Replication can be added as CLI option easily:

https://github.com/apache/ozone/blob/5983ccc0ef66defa8ae27cd09ac0f1ca3471bcb0/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OzoneClientKeyGenerator.java#L80-L81

https://github.com/apache/ozone/blob/5983ccc0ef66defa8ae27cd09ac0f1ca3471bcb0/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OzoneClientKeyGenerator.java#L106

Reply:
Done.